File: Java/Reverse String.java
class Solution {
    public void reverseString(char[] s) {
        int len = s.length;
        boolean isEven = len%2 == 0 ? true : false;
        for (int i = 0; i < len/2; i++) {
            char firstChar = s[i];
            char lastChar = s[len-(i+1)];
            
            s[i] = lastChar;
            s[len-(i+1)] = firstChar;
        }
    }
}

File: Python/Reverse String.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        idx = 0
        while (idx < len(s)/2):
            tmp = s[idx]
            s[idx] = s[-(idx+1)]
            s[-(idx+1)] = tmp
            idx += 1
            
        
